ui/app/css/itcss/components/request-encryption-public-key.scss

```diff
@@ -61,9 +61,9 @@
   }
 
   &__header__text {
+    @include H3;
```

Comment:
font size is between H3 and H4 but closer to H3:

<details>
<summary>Before</summary>

<img src="https://user-images.githubusercontent.com/4448075/88583361-c2d74580-d015-11ea-8bfb-39525aa92503.png" />

</details>
<details>
<summary>After</summary>

<img src="https://user-images.githubusercontent.com/4448075/88583433-dedae700-d015-11ea-8cac-36de8f2b62ce.png" />


</details>